appveyor.yml

```diff
@@ -173,6 +173,7 @@ on_success:
 
 
 on_finish:
+  - appveyor PushArtifact junit-results.xml
```

Comment:
Please add some comment about what this is doing in difference to the UploadFile thing below.

But isn't this duplicate with the UploadFile stuff below?